crmsh/utils.py

```diff
@@ -112,6 +112,8 @@ def user_of(host):
             node = item
         if host == node:
             return user
+    raise ValueError('Failed to get user of host {}'.format(host))
```

Comment:
Many behave test cases blocked here now.
When calling this for the first time, 'core.hosts' may not be set yet.